examples/deployment/scripts/transfer/asc-logic.js

```diff
@@ -0,0 +1,23 @@
+const {
```

Comment:
File name is misleading. Do we transfer ASA's logic in this script? I think not.

Reply:
will  , transfer-check-asc.js be good ?